### PR TITLE
WIP: log JSON interpretation

### PR DESCRIPTION
**Pull request by dannasessha**
WIP.

Currently, this compiles, and passes all tests. It should process `getinfo` and `getblockchaininfo` properly.

However, it does not yet serialized or log these Values.

### CODE REVIEW

**Comment by zancas** [comment]
I think this PR should land when a file is successfully written with a interpreted-serialized-output within!

File: src/main.rs [hunk 1]
```diff
@@ -12,10 +12,14 @@ fn main() {
 
         log_raw_output(command.clone(), raw_command_help.to_string());
 
-        // TODO : make more general and remove `if`
-        if command == "getinfo".to_string() {
+        if command == "getinfo".to_string() || command == "getblockchaininfo".to_string() {
             let interpreted_command_help =
                 quizface::interpret_help_message(raw_command_help);
+            //  write this value, serialized, to file
```
Comment (AloeareV) [inline src/main.rs:18]: Agreed, when this comment is functional code, that would be the ideal time to merge

**Review by zancas — APPROVED** [review]
LGTM

**Comment by dannasessha** [comment]
Woooooo hoooooo!!!

**Pull request merged**